Doc/whatsnew/3.4.rst

```diff
@@ -1528,10 +1528,10 @@ work on Windows.  This change was actually inadvertently made in 3.3.4.
 sunau
 -----
 
-The :meth:`~sunau.getparams` method now returns a namedtuple rather than a
+The :meth:`!~sunau.getparams` method now returns a namedtuple rather than a
```

Comment:
I think these needs flipping:

```suggestion
The :meth:`~!sunau.getparams` method now returns a namedtuple rather than a
```

![image](https://github.com/python/cpython/assets/1324225/731cff9f-9129-4aa8-8e54-a4c136ff291d)


https://cpython-previews--104863.org.readthedocs.build/en/104863/whatsnew/3.4.html#sunau